deps.ts

```diff
@@ -17,7 +17,7 @@ export {
   doc,
   type DocOptions,
   type LoadResponse,
-} from "https://deno.land/x/deno_doc@v0.37.0/mod.ts";
+} from "https://deno.land/x/deno_doc@0.38.0/mod.ts";
```

Comment:
Oops, well we should drop the v prefix anyway 🤷‍♂️

Reply:
I tested the links and dropping the "v" seems to be required now for it to work properly.

Reply:
Yeah, the "oops" was for me not being consistent with the repo and accidentally publishing it without a `v` 😅

(I'm just reviewing this PR to compare it with the new autogenerated PR)